src/rpc/blockchain.cpp

```diff
@@ -529,6 +544,15 @@ static UniValue getrawmempool(const JSONRPCRequest& request)
                         {
                             {RPCResult::Type::OBJ, "transactionid", "", MempoolEntryDescription()},
                         }},
+                    RPCResult{"for verbose = false and mempool_sequence = true",
+                        RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::NUM, "mempool_sequence", "The mempool sequence value."},
+                            {RPCResult::Type::ARR, "txids", "",
```

Comment:
885212e could you please inverse the order of `txids` and `mempool_sequence` fields here in the help to be the same as the rpc output

```diff
                         {
-                            {RPCResult::Type::NUM, "mempool_sequence", "The mempool sequence value."},
                             {RPCResult::Type::ARR, "txids", "",
                             {
                                 {RPCResult::Type::STR_HEX, "", "The transaction id"},
                             }},
+                            {RPCResult::Type::NUM, "mempool_sequence", "The mempool sequence value."},
                         }},
```

Reply:
done